frontend/types/interfaces.ts

```diff
@@ -28,13 +34,30 @@ export interface Step {
   schemaRef: string
 
   render: (RenderInterface) => JSX.Element | null
-  renderBasic?: (RenderInterface) => JSX.Element | null
+  renderBasic: (RenderInterface) => JSX.Element | null
   renderButtons: (RenderButtonsInterface) => JSX.Element | null
 
   shouldValidate: boolean
   isComplete: (step: Step) => boolean
 }
 
+export interface StepNoRender {
+  schema: any
```

Comment:
I know there's a ticket to type `schema`, but can we type `uiSchema` or `state` any further?

Reply:
uischema seems to have a type but struggling for state.